lib/citgm.js

```diff
@@ -4,6 +4,12 @@
 var util = require('util');
 var EventEmitter = require('events');
 
+// This is a shim to allow for v0.10 to work
+// TODO remove when we no longer support v0.10
+if (EventEmitter.EventEmitter) {
```

Comment:
This `if` shouldn't be necessary: https://github.com/nodejs/node/commit/6ed861dd7feea33224d1cac8e64010b45974ae51